cmd/scollector/collectors/disk.go

```diff
@@ -0,0 +1,141 @@
+package collectors
+
+// things that are not OS specific.
+// ie: can compile and execute anywhere
+
+import (
+	"bufio"
+	"io"
+	"strconv"
+	"strings"
+
+	"bosun.org/metadata"
+	"bosun.org/opentsdb"
+)
+
+// Check mdadm raid arrays on linux
+// linux.disk.mdadm.state values:
+const (
+	mdadmUnknown  = iota
+	mdadmNormal   // 1 => active or clean state
```

Comment:
If these are based on linux.disk.madm values, don't use iota.  iota could change if someone inserts an item in the middle. If this is a copy of constants used by linux.disk.mdadm.state, set them to the constant values.